### PR TITLE
QuantizedDot and QuantizedDotBias ops CPU impl

### PR DESCRIPTION
**Pull request by ashokei**

### CODE REVIEW

**Comment by diyessi** [comment]
#2593 is a cleaned up replacement.

**Comment by diyessi** [comment]
Clicked on wrong line.

**Comment by nbpatel** [comment]
where is the scale getting computed? Also, don't see quantization.cpp

**Comment by ashokei** [comment]
@nbpatel similar to QuantizedConv , the scale is computed in the builders. 
Added builders, scale computation, and unittests.

File: src/ngraph/runtime/cpu/CMakeLists.txt [hunk 1]
```diff
@@ -42,8 +42,7 @@ set(SRC
     builder/concat.cpp
     builder/convert.cpp
     builder/convert_layout.cpp
-    builder/quantized_conv.cpp
-    builder/quantized_concat.cpp
+    builder/quantized_dot.cpp
```
Comment (silee2) [inline src/ngraph/runtime/cpu/CMakeLists.txt]: Remove this line. It also shows up at line 66.

**Pull request merged**